platform/ios/detect.py

```diff
@@ -140,7 +140,6 @@ def configure(env: "SConsEnvironment"):
             )
         )
         env.Append(ASFLAGS=["-arch", "arm64"])
-        env.Append(CPPDEFINES=["NEED_LONG_INT"])
```

Comment:
This might be used elsewhere, unlikely perhaps but it might be best to leave it, doesn't really hurt and would prevent breaking custom modules

Reply:
It's not used anywhere else in the repo itself, but the module point is fair. I'll preserve it and append a note instead.